nvidia_triton/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,32 @@
+name: Nvidia Triton
+files:
+- name: nvidia_triton.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/openmetrics
+      overrides:
+        openmetrics_endpoint.required: true
+        openmetrics_endpoint.hidden: false
+        openmetrics_endpoint.display_priority: 2
+        openmetrics_endpoint.value.example: http://localhost:8002/metrics
+        openmetrics_endpoint.description: |
+          Endpoint exposing the Nvidia Triton's Prometheus metrics. For more information refer to:
+          https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/metrics.html#metrics
+    - name: collect_server_info
+      display_priority: 1
+      description: |
+        Weather to collect server status and metadata from HTTP Endpoint of Nvidia Triton API.
+      value:
+        display_default: true
+        example: true
+        type: boolean
+    - name: server_port
+      description: |
+        The port exposing HTTP Endpoint of Nvidia Triton API.
+      value:
+        display_default: "8000"
+        type: string
```

Comment:
Any reasons to use a string instead of an integer here?

Reply:
It will be appended to a string to construct the URL. But it is possible to just convert it to a string in the code.